src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+(() => {
+  const body = document.querySelector('body');
+
+  document.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+  });
+
+  const notification = (parentNode, classes, text) => {
+    const node = document.createElement('div');
+
+    node.classList.add(...classes);
+    node.textContent = text;
+    node.dataset.qa = 'notification';
+    parentNode.append(node);
+  };
```

Comment:
This is a code quality issue. The 'notification' function is tightly coupled with the 'body' element. This makes the function less flexible and harder to test. Consider passing the parent node as a parameter to the function, so it can append the notification to any given node.